openlibrary/catalog/edition_merge/templates/index.html

```diff
@@ -1,61 +1,69 @@
 <html>
+
 <head>
-<title>Merge duplicate editions</title>
-<style>
-td { vertical-align: top; background: #ddd };
-</style>
+  <title>Merge duplicate editions</title>
+  <style>
+    td {
+      vertical-align: top;
+      background: #ddd
+    };
+  </style>
 </head>
+
 <body>
-<h1>Merge duplicate editions</h1>
+  <h1>Merge duplicate editions</h1>
 
-{% with messages = get_flashed_messages() %}
+  {% with messages = get_flashed_messages() %}
```

Comment:
These indentation changes don't seem related. Can you explain what they are for in the pull request description or remove them?